modify readme.md file

Tshepo45 opened the pull request:
-edit readme.md file

tufoinnkuo10 reviewed (changes requested):
Hi @Tshepo45 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has its node_module file in your remote repository. please remove it, add it to the gitignore file and ensure it doesn't exist in your remote repository. 😄 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

sja-thedude reviewed (commented):
## Invalid Code Review Request ❌

Hi @Tshepo45, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have a meaningful Pull Request title and description. Kindly make sure to add those before re-submitting.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

Tshepo45 commented:
hi ***@***.*** *I have followed the instructions and removed the
node_module file and added it into the .gitignore file.could you please
clarify more what i am missing.


On Thu, May 26, 2022 at 10:38 AM Syeda Juveria Afreen <
***@***.***> wrote:

> ***@***.**** commented on this pull request.
> Invalid Code Review Request ❌
>
> Hi @Tshepo45 <https://github.com/Tshepo45>, This is @sja-thedude
> <https://github.com/sja-thedude> your code reviewer.
>
> While you made a great effort in this project, unfortunately, I cannot
> proceed to review your code.
>
> You have submitted a project, that doesn't have a meaningful Pull Request
> title and description. Kindly make sure to add those before re-submitting.
>
> Your Code Review Request will be marked as invalid in your Dashboard, so
> please submit a new one once you are ready 🙏
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me ***@***.*** <https://github.com/sja-thedude>]
> in your question so I can receive the notification.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Tshepo45/Portfolio-Mobile/pull/1#pullrequestreview-985907189>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXHWXRJ7K4MNUTFLVXMVUMTVL42CDANCNFSM5W44EVRQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

arslanbisharat commented on README.md:
- [ ] Please add node modules to the gitignore.

arslanbisharat commented on index.html:
- [ ] The hamburger button should be on the right and has a nav tag

arslanbisharat reviewed (changes requested):
Hi @tufoinnkuo10, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
 - [ ] You have added description to the PR but that doesn't meet the standards. Please add proper description in there. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

danielufeli commented on README.md:
- [ ] Like the previous reviewer said kindly add node_modules/ folder to your .gitignore file, to add a folder in the .gitignore file you need to add a / to the folder name e.g. `node_modules/`

danielufeli reviewed (changes requested):
Hi @Tshepo45 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Clean Design ✔️
- Passing linters ✔️
- Descriptive Readme ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

phelian23 commented on .gitignore:
- [ ] Good job adding the node_modules to the .gitignore file as requested by the previous reviewer. However, it's still in your repository since you've already committed and pushed it before adding it to the .gitignore file, therefore you'll have to manually delete from the repository. Kindly do that

phelian23 commented on index.html:
- [ ] [Optional] Kindly wrap the logo in an `<a>`, it is a good practice to wrap the logo in an `<a>` as this will serve as a link to the home from anywhere on the page.
- [ ] [Optional] Also all the images are not displaying, as the image folder does not exist in your repository. Kindly add it and the image files

phelian23 commented on index.html:
- [ ] [Optional] This image file doesn't exist in your repository, kindly add it.

phelian23 reviewed (changes requested):
Hi @Tshepo45, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Tshepo45 commented on .gitignore:
hi phelian23 I have deleted it manually from the repository ,do u mean I should also delete the one in the .gitignore file

Kandy-Peter commented on .gitignore:
- [ ] Rather, delete it from the Github repo. As you can notice, it still present.

Kandy-Peter commented on index.html:
- [ ] As suggested by the previous reviewer, you have to import those images into your project folder with the correct path and name. Please do that so they can be displayed on the view.

Kandy-Peter commented on index.html:
- [ ] OPTIONAL: Indent your code to make it professional and readable.

Kandy-Peter reviewed (changes requested):
Hi @Tshepo45 ,

Great work on making the changes requested by a previous reviewer 👏
You've done well implementing some of the requested changes, but there are still some issues which aren't been addressed yet.

![image](https://user-images.githubusercontent.com/80612925/169586832-56418ba8-59ea-4376-97ab-65fb0f51c869.png)

## TO HIGHLIGHT:  
- Well structured files 💯 
- Good commit history :heavy_check_mark: 

## Required Changes ♻️
- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)..

Ginohmk commented on index.html:
- Please structure your HTML tags properly with the right CSS, so as to get similar looks with the Figma design

##### Your design
![Screenshot from 2022-05-26 19-54-16](https://user-images.githubusercontent.com/58771507/170558625-c437c5b7-b293-4c8e-9185-7ca4e40346e1.png)

##### Figma design 
![Screenshot from 2022-05-26 20-03-51](https://user-images.githubusercontent.com/58771507/170558794-9e89ec16-ad81-4a78-a468-76892e9c0d4f.png)

Please feel free to reach out to me, if you find this feedback confusing in anyway :+1:

Ginohmk reviewed (changes requested):
Hi  @Tshepo45 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![keep-trying-youre-almost-there](https://user-images.githubusercontent.com/67087664/169483884-c624db70-793d-42be-8ebe-afef6a1cd5c3.gif)

 ## Highlights
- Linters passed :100: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

#### You can reach the code reviewer 
##### Slack: kanu mike chibundu

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Tshepo45 commented:
- Please the proper way to use a <ul> tag is as follow

<ul>

 <li>  <img src="/image/Ellipse2.png" alt="ellipse2"> </li>

 <li>  <img src="/image/Ellipse2.png" alt="ellipse2"> </li>

</ul>

So how do I access different items for positioning in css if I
structure my <ul> in this way?


On Thu, May 26, 2022 at 9:34 PM tshepo makama ***@***.***>
wrote:

> This feedback is really confusing and frustrating.I have had 5 reviewers
> all with different changes that I did my best implementing.its like my
> progress is being halted .can't a reviewer identify all the changes
> required once and not bit by bit from different reviewers.i have attached
> my design from my side i do not know how when it get to the reviewer is not
> the same.[image: image.png]
>
>
> On Thu, May 26, 2022 at 9:07 PM Kanu Mike Chibundu <
> ***@***.***> wrote:
>
>> ***@***.**** requested changes on this pull request.
>>
>> Hi @Tshepo45 <https://github.com/Tshepo45> ,
>>
>> Great work on making the changes requested by a previous reviewer 👏
>> You've done well implementing some of the requested changes, but there
>> are still some which aren't addressed yet.
>>
>> [image: keep-trying-youre-almost-there]
>> <https://user-images.githubusercontent.com/67087664/169483884-c624db70-793d-42be-8ebe-afef6a1cd5c3.gif>
>> Highlights
>>
>>    - Linters passed 💯
>>
>> Required Changes ♻️
>>
>> *Check the comments under the review.*
>> Optional suggestions
>>
>> *Every comment with the [OPTIONAL] prefix won't stop the approval of this
>> PR. However, I strongly recommend you to take them into account as they can
>> make your code better. Some of them were simply missed by the previous
>> reviewer and addressing them will really improve your application.*
>> You can reach the code reviewer Slack: kanu mike chibundu
>>
>> Cheers and Happy coding!👏👏👏
>>
>> Feel free to leave any questions or comments in the PR thread if
>> something is not 100% clear.
>> Please, remember to tag me in your question so I can receive the
>> notification.
>>
>> *Please, do not open a new Pull Request for re-reviews. You should use
>> the same Pull Request submitted for the first review, either valid or
>> invalid unless it is requested otherwise.*
>> ------------------------------
>>
>> *As described in the Code reviews limits policy
>> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
>> limited number of reviews per project (check the exact number in your
>> Dashboard). If you think that the code review was not fair, you can request
>> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
>> ------------------------------
>>
>> In index.html
>> <https://github.com/Tshepo45/Portfolio-Mobile/pull/1#discussion_r882980918>
>> :
>>
>> > +        <ul class="d">
>>
>>           <li>
>>
>>            <img src="/image/Ellipse2.png" alt="ellipse2">
>>
>>           </li>
>>
>>
>>
>>    - Please the proper way to use a <ul> tag is as follow
>>
>> <ul>
>>
>>  <li>  <img src="/image/Ellipse2.png" alt="ellipse2"> </li>
>>
>>  <li>  <img src="/image/Ellipse2.png" alt="ellipse2"> </li>
>> </ul>
>>
>> For more info on list click here
>> <https://www.w3schools.com/html/html_lists.asp>
>> ------------------------------
>>
>> In index.html
>> <https://github.com/Tshepo45/Portfolio-Mobile/pull/1#discussion_r882985013>
>> :
>>
>> > +<header class="header-section">
>>
>> +
>>
>> +  <nav>
>>
>> +      <img class="my-logo" src="/image/logo.png" alt="my logo">
>>
>> +        <ul class="u">
>>
>> +          <li>
>>
>> +            <img src="/image/Union.png" alt="ellipse">
>>
>> +          </li>
>>
>> +        </ul>
>>
>> +  </nav>
>>
>> +
>>
>> + <!--vectors and ellipse top section-->
>>
>> +  <div>
>>
>> +        <ul class="r">
>>
>>
>>
>>    - Please structure your HTML tags properly with the right CSS, so as
>>    to get similar looks with the Figma design
>>
>> Your design
>>
>> [image: Screenshot from 2022-05-26 19-54-16]
>> <https://user-images.githubusercontent.com/58771507/170558625-c437c5b7-b293-4c8e-9185-7ca4e40346e1.png>
>> Figma design
>>
>> [image: Screenshot from 2022-05-26 20-03-51]
>> <https://user-images.githubusercontent.com/58771507/170558794-9e89ec16-ad81-4a78-a468-76892e9c0d4f.png>
>>
>> Please feel free to reach out to me, if you find this feedback confusing
>> in anyway 👍
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/Tshepo45/Portfolio-Mobile/pull/1#pullrequestreview-986658575>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AXHWXRIFKEBQOENYVTSKLKTVL7DW7ANCNFSM5W44EVRQ>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>

Ginohmk commented:
- That can be done like so.. give the ul a class and then you give that class in the CSS a display of flex . This will align all li item to be rows, then to access each li items, you can use this in your css .. the class name for the ul tag followed by li .. 

Let me write in code below. 

```html
<ul class = 'social_icon_container'>
<li>  <img src="/image/Ellipse2.png" alt="ellipse2"> </li>

<li>  <img src="/image/Ellipse2.png" alt="ellipse2"> </li>
</ul>
```

- In CSS 
```css
// This makes the li items a row and adds spaces //between them with the use of flex gap
.social_icon_container {
display: flex;
gap:20px;
}

// this access all li item and removes the bullet dot
.social_icon_container li {
list-style-type: none;
}
```

- if you still find it confuses, let me know so I explain further

Mre55 commented on index.html:
**[REQUIRED CHANGE]** As mentioned by previous reviewers and emphasized by the last reviewer the `ul` should be one and you have to use `li` inside that `ul` currently you have more than 6 `ul` which is not the proper use of `ul`. Please refer below to the proper use of `ul` you already implement for one of your elements for the social media links 👇 

<img width="907" alt="image" src="https://user-images.githubusercontent.com/34578631/170580015-0b51c1b5-daa8-4650-bdf4-d03e3187ce0a.png">

Mre55 commented on index.html:
**[OPTIONAL]** Rather than using this many `ul` and `li` for the background image I highly recommend using this `.png` which you can get from the Figma design provided.

![header-shapes mobile](https://user-images.githubusercontent.com/34578631/170580774-affdde92-88b0-4ae3-9d6a-f9ec184358c9.png)

Mre55 reviewed (changes requested):
**Hi @Tshepo45,**

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights 🎯

- The Pull Request is following the GitHub Flow ✅ 
- Linters are working properly ✅ 

 ## Required Changes ♻️
_Check the comments under the review._

- [ ] As requested by the previous reviewers you pull request description is not giving short summary of what you did in this project. You can find [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects) how to do it.

<img width="430" src="https://media1.giphy.com/media/oVjOd9TEZDmSTc4x37/giphy.gif"/>

## Optional suggestions 🟡 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

mke2111 commented:
Project Under Second Opinion Review.
Kindly wait for a response. 😄

mke2111 reviewed (commented):
Hello @Tshepo45 👋,

_This is an additional review as a result of the second opinion request._

Great work on your first project. 💯 

The request from the last CR is appropriate. The best HTML and CSS practices use `li` tags inside one `ul` tag that are then considered to be grouped together. I suggest you make this change where necessary.
However, since this request was not clearly stated by the previous reviewers especially right from the first review, I will consider this request optional.

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please note that since we will approve it manually so this could take a few hours!

Keep rocking 🚀!

Tshepo45 committed:
modify readme.md file

Tshepo45 committed:
add .gitignore file

Tshepo45 committed:
delete node-modules file

Tshepo45 committed:
delete file
